src/lib/util.ts

```diff
@@ -292,20 +292,47 @@ export function transformDiscordLocale(discordLocale: string): string {
 	else return "en"
 }
 
-export function updateButtonColors(row: MessageActionRow, page: number, pages: unknown[]) {
-	if (page === 0) {
-		row.components.forEach(button => {
-			if (button.customId === "first" || button.customId === "previous") (button as MessageButton).setStyle("SECONDARY").setDisabled(true)
-			else (button as MessageButton).setStyle("SUCCESS").setDisabled(false)
-		})
-	} else if (page === pages.length - 1) {
-		row.components.forEach(button => {
-			if (button.customId === "last" || button.customId === "next") (button as MessageButton).setStyle("SECONDARY").setDisabled(true)
-			else (button as MessageButton).setStyle("SUCCESS").setDisabled(false)
-		})
-	} else row.components.forEach(button => (button as MessageButton).setStyle("SUCCESS").setDisabled(false))
-
-	return row
+export function createButtonControls(
+	pageIndex: number,
+	pages: unknown[],
+	options: { localizations?: GetStringFunction; itemName?: string } = { itemName: "page" },
```

Comment:
The name of the function should stay consistent (other references should be fixed accordingly)
```suggestion
	options: { getString?: GetStringFunction; itemName?: string } = { itemName: "page" },
```